docs/interfaces.rst

```diff
@@ -220,4 +164,4 @@ If you want to do an external call to another contract, Vyper provides an extern
         def forwardWeight(delegate_with_weight_to_forward: address): nonpayable
         # ...
 
-The output can then easily be copy-pasted to be consumed.
+The output can then easily be copy-pasted directly in a regular vyper file.
```

Comment:
or imported?

Reply:
i mean the output is text, so you would have to put it into a file somehow